Services/AuthentificationService/Controllers/AuthentificationController.cs

```diff
@@ -66,6 +72,49 @@ public string GetToken()
         [Route("[controller]/check")]
         [HttpGet]
         public string CheckToken(string token)
-            => tokens.Contains(token).ToString();
+        {
+            var index = BinarySearch(token);
+            int compare = String.Compare(TokensStorage.Tokens[index], token);
+            if (compare != 0)
+            {
+                return "false";
+            }
+            return "true";
```

Comment:
How about `return compare == 0;`? Why do we need to return `string` instead of `bool`?

Reply:
Is this method going to work at all?

Reply:
> How about `return compare == 0;`? Why do we need to return `string` instead of `bool`?


This was done in order to test the operation of methods

Reply:
> Is this method going to work at all?

It correctly determines if a token exists